cmd/fluxctl/release_cmd.go

```diff
@@ -179,6 +187,53 @@ func (opts *controllerReleaseOpts) RunE(cmd *cobra.Command, args []string) error
 
 		opts.dryRun = false
 	}
+	if opts.watch {
+		await(ctx, cmd.OutOrStdout(), cmd.OutOrStderr(), opts.API, jobID, !opts.dryRun, opts.verbosity)
+
+		fmt.Fprintf(cmd.OutOrStderr(), "Monitoring rollout...\n")
```

Comment:
In other places we have whitespace before the triple dots, looks a bit off when appearing after each other
```
Submitting release ...
Monitoring rollout...
```

Reply:
at the moment both styles are used in the command - i'll update so that there's always a whitespace before the dots.